src/store/trails/actions.js

```diff
@@ -193,6 +195,49 @@ export const addBallot = async function ({ commit, state, rootState }, ballot) {
         }
       }
     ]
+    if (ballot.treasurySymbol.symbol === 'VOTE') {
+      togglebal = {
+        account: 'telos.decide',
+        name: 'togglebal',
+        data: {
+          ballot_name: ballotName,
+          setting_name: 'votestake'
+        }
+      }
+    } else if (!ballot.settings) {
+      togglebal = {
+        account: 'telos.decide',
+        name: 'togglebal',
+        data: {
+          ballot_name: ballotName,
+          setting_name: 'voteliquid'
+        }
+      }
+    } else if (ballot.settings && ballot.config) {
+      if (ballot.config === 'both') {
+        for (let i of ['voteliquid', 'votestake']) {
+          togglebal = {
+            account: 'telos.decide',
+            name: 'togglebal',
+            data: {
+              ballot_name: ballotName,
+              setting_name: i
+            }
+          }
+        }
+      } else {
+        togglebal = {
+          account: 'telos.decide',
+          name: 'togglebal',
+          data: {
+            ballot_name: ballotName,
+            setting_name: ballot.config
+          }
```

Comment:
It looks like all of the `togglebal ` objects are the same except for `setting_name`, you _could_ add a method to return that object passing that value.